CONTRIBUTING.md

```diff
@@ -13,7 +13,7 @@ If your repo has certain guidelines for contribution, put them here ahead of the
 -->
 
 - [Contributor License Agreement](https://git.k8s.io/community/CLA.md) Kubernetes projects require that you sign a Contributor License Agreement (CLA) before we can accept your pull requests
-- [Kubernetes Contributor Guide](http://git.k8s.io/community/contributors/guide) - Main contributor documentation, or you can just jump directly to the [contributing section](http://git.k8s.io/community/contributors/guide#contributing)
+- [Kubernetes Contributor Guide](https://github.com/kubernetes/community/tree/master/contributors/guide) - Main contributor documentation, or you can just jump directly to the [contributing section](http://git.k8s.io/community/contributors/guide#contributing)
```

Comment:
```suggestion
- [Kubernetes Contributor Guide](https://git.k8s.io/community/contributors/guide/README.md) - Main contributor documentation, or you can just jump directly to the [contributing section](https://git.k8s.io/community/contributors/guide/README.md#contributing)
```

Reply:
No problem.

Reply:
@wangxy518 -- did you have a chance to accept this suggestion? It would need to be committed for me to approve this PR.